app/controllers/api/v1/customers_controller.rb

```diff
@@ -70,6 +70,24 @@ def destroy
         end
       end
 
+      def checkout_url
+        customer = current_organization.customers.find_by(external_id: params[:customer_external_id])
+
+        result = ::Customers::GenerateCheckoutUrlService.call(customer:)
+
+        if result.success?
+          render(
+            json: ::V1::PaymentProviders::CustomerCheckoutSerializer.new(
+              customer,
+              root_name: 'customer',
+              checkout_url:result.checkout_url,
```

Comment:
Layout/HashAlignment: Align the keys of a hash literal if they span more than one line.